### PR TITLE
Screens Added: Python file of screens are added

### PR DESCRIPTION
**Pull request by ikramali585**

### CODE REVIEW

**Review by M-Usman-91 — CHANGES_REQUESTED** [review]
### Write down a reason...
- Why did you remove the complete conetnts?
### Expected changes
- Revert the contents...

File: Authentication/LoginRecord.csv [hunk 1]
```diff
@@ -1 +1 @@
-Name,Time,Date
+
```
Comment (M-Usman-91) [inline Authentication/LoginRecord.csv]:
```suggestion
Name,Time,Date

```

**Review by M-Usman-91 — COMMENTED** [review]
It won't be readable if the header file is deleted...

**Comment by ikramali585** [comment]
I did it mistakenly
How can I revert it?

File: Authentication/LoginRecord.csv [hunk 2]
```diff
@@ -1 +1,7 @@
 Name,Time,Date
+2019-CE-34,22:55:31,11-12-21
+2019-CE-34,22:58:22,11-12-21
+WIN_20211212_00_18_34_Pro,00:19:31,12-12-21
+2019-CE-41,00:22:46,12-12-21
+2019-CE-41,16:16:14,12-12-21
+2019-CE-41,16:17:12,12-12-21
```
Comment (M-Usman-91) [inline Authentication/LoginRecord.csv]: ### Changes Required...
- Only the first line was required...
- Push it with that commit...

**Review by M-Usman-91 — CHANGES_REQUESTED** [review]
Change the code according to the following request....

**Review by M-Usman-91 — APPROVED** [review]
Approving Views

**Pull request merged**